src/apps/competitions/models.py

```diff
@@ -352,6 +353,15 @@ def check_future_phase_submissions(self):
             logger.info(f"This competition is missing the previous phase to migrate from.")
 
 
+class TaskOrder(models.Model):
+    task = models.ForeignKey(Task, on_delete=models.CASCADE)
+    phase = models.ForeignKey(Phase, on_delete=models.CASCADE)
+    order_index = models.PositiveIntegerField(default=999)
```

Comment:
You can add some `class Meta:` info to this class so by default it sorts by order, that way you don't need special serializer stuff